modules/data-loading/partials/create-connector.adoc

```diff
@@ -0,0 +1,13 @@
+=== Create connector
+Once you have prepared the configuration file, run command `gadmin connector create` and specify the configuration file to create the connector:
+
+[.wrap,console]
+----
+gadmin connector create --c <config_file>
+----
+
+The connectors start streaming from the data source immediately after creation if the configurations are valid.
+You can run `gadmin connector status` to verify the status of the connectors.
+If the configurations are valid, the connectors should be in `RUNNING` status.
+
+Data streamed from the source stays in the destination cluster Kafka topics until they are ingested by a loading job.
```

Comment:
Do we plan to add more details on how our Pruner works? Will that be in a separate doc? We need to tell users about prune interval and some known issues for pruners.